src/test/spec/crud/aggregate.rs

```diff
@@ -62,9 +62,14 @@ async fn run_aggregate_test(test_file: TestFile) {
                 .await
                 .expect(&test_case.description);
 
+            let results = cursor
+                .try_collect::<Vec<_>>()
+                .await
+                .expect(&test_case.description);
+
             assert_eq!(
                 outcome.result.unwrap_or_default(),
-                cursor.try_collect::<Vec<_>>().await.unwrap(),
+                results,
```

Comment:
During my attempts to fix the issue, this test failed, and the use of `unwrap` made it difficult to determine which test was failing, so I updated it to use `expect`.